service/web/subscriberserver.go

```diff
@@ -229,6 +229,11 @@ func (s subscriberserver) ModifyAckDeadline(ctx context.Context, req *metrov1.Mo
 	return new(emptypb.Empty), nil
 }
 
-func (s subscriberserver) AuthFuncOverride(ctx context.Context, _ string) (context.Context, error) {
-	return interceptors.AppAuth(ctx, s.credentialCore)
+//AuthFuncOverride - Override function called by the auth interceptor
+func (s subscriberserver) AuthFuncOverride(ctx context.Context, _ string, req interface{}) (context.Context, error) {
+	projectID, err := getProjectIDFromRequest(ctx, req)
```

Comment:
we are duplicating these lines in each server for now. Let's try and make this common. Maybe wrap `interceptors.AppAuth(ctx, s.credentialCore, projectID)` in another func call and call that from each server impl.